docs/canvas/canvas-common-functions.asciidoc

```diff
@@ -6,640 +6,765 @@ The common functions can run anywhere, which means they'll execute wherever
 the expression is currently executing. For example, if the engine is currently 
 running on the server, the functions will run on the server.
 
-A *** denotes a required argument.
+A *** denotes a required argument. 
 
+A *†* denotes an argument can be passed multiple times.
 
 [float]
-=== all
+[[all_fn]]
+=== `all`
 
-Returns true if all of the conditions are true.
+Returns `true` if all of the conditions are `true`. See also <<any_fn>>.
 
-*Accepts:* Anything or nothing
+*Accepts:* `null`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_*
-|boolean
+|_Unnamed_ *** *†*
+
+Alias: `condition`
+|`boolean`
 |The conditions to check
 |===
 
-*Returns:* boolean
+*Returns:* `boolean`
 
 
 [float]
-=== alterColumn
+[[alterColumn_fn]]
+=== `alterColumn`
 
-Converts between core types, for example, string, number, null, boolean, date 
-and rename columns.
+Converts between core types, for example, `string`, `number`, `null`, `boolean`, `date` and renames columns. See also <<mapColumn_fn>> and <<staticColumn_fn>>.
 
-*Accepts:* datatable
+*Accepts:* `datatable`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|column*
-|string
+|_Unnamed_ *** 
+
+Alias: `column`
+|`string`
 |The name of the column to alter
 
-|type
-|string
-|The type to convert the column to; leave blank to not change type (default: `null`)
+|`type`
+|`string`
+|The type to convert the column to; leave blank to not change type
 
-|name
-|string
-|The resultant column name; leave blank to not rename (default: `null`)
+|`name`
+|`string`
+|The resultant column name; leave blank to not rename
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== any
+[[any_fn]]
+=== `any`
 
-Returns true if any of the conditions are true.
+Returns `true` if any of the conditions are `true`. See also <<all_fn>>.
```

Comment:
How about:

When all of the conditions are `true`, returns `true`. See also <<all_fn>>.